localstack-core/localstack/utils/files.py

```diff
@@ -80,9 +81,14 @@ def _opener(path, flags):
         f.flush()
 
 
-def load_file(file_path: str, default=None, mode=None):
+def load_file(
+    file_path: str | os.PathLike, default: Any = None, mode: str | None = None, strict: bool = False
```

Comment:
```suggestion
    file_path: str | os.PathLike, default: str | bytes | None = None, mode: str | None = None, strict: bool = False
```
I think default should really only be str or bytes.